### DISCUSSION TITLE
使用outlook365邮箱，SMTP服务身份认证不成功

### DISCUSSION THREAD

**Discussion by Andy-njupt** [walinejs/waline#1878] [Q&A]
报错内容：535 5.7.139 Authentication unsuccessful, the request did not meet the criteria to be authenticated successfully. Contact your administrator。
我的配置：
SMTP_USER：邮箱账号
SMTP_PASS：邮箱密码
SITE_NAME：'doc'
SMTP_SERVICE: 'Outlook365'
我不确定是outlook365身份认证方式变动了，还是我的配置有问题？帮忙看下呗 感谢感谢

**Comment by hui-shao** [comment]
这个问题我遇到过，主要是因为开启了两步验证。
可以去微软账户安全中心获取一个应用密码来作为 smtp 登录密码。
https://account.live.com/proofs/Manage/additional

**Reply by Andy-njupt** [reply]
谢谢您，您发的这个地址是否适用于企业账户，我利用企业outlook365邮箱无法进行登录

**Reply by hui-shao** [reply]
> 谢谢您，您发的这个地址是否适用于企业账户，我利用企业outlook365邮箱无法进行登录

企业的我就不太了解了。
不过可以试试看？
大概率是这个双重认证的问题

**Reply by Andy-njupt** [reply]
好的，感谢
